client/src/pages/AddBusiness/AddBusiness.tsx

```diff
@@ -247,20 +250,19 @@ export const AddBusiness = () => {
                     height='100px'
                   />
                   <label htmlFor='select-image'>
-                    <Button
```

Comment:
Hi @cynthiaeddy! Per our conversation, currently, without the `Button` from '@material-ui/core/Button,' the upload image new widow popover functionality breaks. I think we should find an alternative working solution before removing this dependency.

Reply:
hi @CodeMeKathy! I've updated the AddBusiness component to mirror the UserRegistration component as far as adding an image in order to remove material/ui.